crypto/engine/eng_cryptodev.c

```diff
@@ -213,29 +222,31 @@ static struct {
 static struct {
     int id;
     int nid;
-    int keylen;
+    int digestlen;
 } digests[] = {
+#  if 0
+        /* HMAC is not supported */
     {
         CRYPTO_MD5_HMAC, NID_hmacWithMD5, 16
     },
     {
         CRYPTO_SHA1_HMAC, NID_hmacWithSHA1, 20
     },
     {
-        CRYPTO_RIPEMD160_HMAC, NID_ripemd160, 16
-        /* ? */
+        CRYPTO_MD5, NID_md5, 16
     },
     {
-        CRYPTO_MD5_KPDK, NID_undef, 0
+        CRYPTO_SHA1, NID_sha1, 20
     },
     {
-        CRYPTO_SHA1_KPDK, NID_undef, 0
+        CRYPTO_SHA2_256, NID_sha256, 32
     },
+#  endif
```

Comment:
Or oh, does "HMAC is not supported" mean it isn't *in this file*?  Then I can understand, and I suppose the `# if 0`d section is only temporarily killed, is that correct?